lib/kubernetes-deploy/runner.rb

```diff
@@ -55,11 +55,12 @@ def self.with_friendly_errors
       exit 1
     end
 
-    def initialize(namespace:, environment:, current_sha:, context:, wait_for_completion:, template_folder: nil)
+    def initialize(namespace:, current_sha:, context:, wait_for_completion:, template_dir:)
       @namespace = namespace
       @context = context
       @current_sha = current_sha
-      @template_path = File.expand_path('./' + (template_folder || "config/deploy/#{environment}"))
+
+      @template_dir = Pathname.new(template_dir).realpath
```

Comment:
What is the difference between `Pathname.new(..)` and `File.expand_path`?

```ruby
irb(main):007:0> Pathname.new("~/.bashrc").realpath
Errno::ENOENT: No such file or directory @ realpath_rec - /Users/sirup/~
	from (irb):7:in `realpath'
	from (irb):7:in `realpath'
	from (irb):7
	from /opt/rubies/2.3.3/bin/irb:11:in `<main>'
irb(main):008:0> File.expand_path("~/.bashrc")
=> "/Users/sirup/.bashrc"
```

In this I feel `File.expand_path` wins 🤔  But for anything not involving `~`, it seems to work the same.